File: composeApp/src/androidMain/kotlin/com/snapchef/app/features/groups/presentation/RecipeGroup.kt
package com.snapchef.app.features.groups.presentation

data class GroupMember(
    val username: String,
    val avatarSeed: String = username,
    val id: Int? = null,
)

data class RecipeGroup(
    val id: String,
    val name: String,
    val code: String?,
    val recipes: List<SharedRecipe>,
    val ownerUsername: String? = null,
    val members: List<GroupMember> = emptyList(),
    val isPersonal: Boolean = false,
)

File: iosApp/iosApp/ui/RecipesScreen/RecipeGroup.swift
//
//  RecipeGroup.swift
//  iosApp
//
//  Created by gergana on 3/27/26.
//

import Foundation

struct RecipeGroup: Identifiable, Equatable {
    let id: String
    let name: String
    let code: String?
    var recipes: [SharedRecipe]
    var isPersonal: Bool = false
}
